Source/UserManagement/Domain/MessageGenerator/MessageGenerator.cs

```diff
@@ -1,10 +1,8 @@
 using doLittle.Domain;
 using Events;
 using System;
-using System.Collections.Generic;
-using System.Text;
 
-namespace Domain.MessageGenerators
+namespace Domain.MessageGenerator
```

Comment:
I'm not sure MessageGenerator is an Aggregate Root.  I think it might be needed just now for technical reasons (I don't think policies can publish events).